.circleci/config.yml

```diff
@@ -622,6 +647,7 @@ workflows:
       - build-buster-securedrop-workstation-grsec
       - build-buster-securedrop-workstation-config
       - build-buster-securedrop-keyring
+      - build-bullseye-securedrop-client
```

Comment:
I think this is the core of what we're trying to do, and because of the dependency chain ti also caused nighltlies to be built :white_check_mark: